src/main/java/pl/winiarczyk/CinemaEmployees/Manager.java

```diff
@@ -0,0 +1,24 @@
+package pl.winiarczyk.CinemaEmployees;
+
+public class Manager extends BaseEmployee {
+    final int salary = 5000;
+
+    public int getBonus() {
```

Comment:
ten getter wydaje się zbędny

Reply:
setter był potrzebny, ale to nie oznacza, że od razu musimy gettera wystawiac ;)